### DISCUSSION TITLE
A secure channel could no be established: Could not create SSL/TLS secure channel.

### DISCUSSION THREAD

**Discussion by antonioaayala** [microsoft/ALAppExtensions#21733] [General]
Hello,

I have an issue calling API from AL on SaaS, when I deploy my app on docker Onpremise container Client.Send works correctly, but whe i deploy to production or Sandox on SaaS, I got the error "Could not create SSL/TLS secure channel...."

My code is

var
    CB64: Codeunit "Base64 Convert";
    Client: HttpClient;

    Request: HttpRequestMessage;
    Content: HttpContent;
    Response: HttpResponseMessage;
    RequestHeaders: HttpHeaders;
    ContentHeaders: HttpHeaders;
    CertString: Text;
    ContentString: Text;
    ResponseString: Text;
    AuthString: Text;

begin
    Clear(Client);

    Request.SetRequestUri('https');
    Request.Method := 'GET';

    AuthString := 'Basic xxxxxxxxxxxxxxxxxx';
    Client.DefaultRequestHeaders.Add('Authorization', AuthString);
    Client.DefaultRequestHeaders.Add('User-Agent', 'Dynamics 365');
    Client.Get('https', Response);

    Response.Content.ReadAs(ResponseString);
    Message(Format(Response.HttpStatusCode));
    message(ResponseString);
end;
I have tried wiht Client.Send to

Thanks a lot

**Comment by JesperSchulz** [comment]
This repo is mostly for extensibility issues. I think you'll have better chances of getting this question answered by the community in other channels (such as Yammer).
